parent/pom.xml

```diff
@@ -297,7 +297,7 @@
                 <pluginRepository>
                     <id>sonatype-nexus-staging</id>
                     <name>Sonatype Nexus Staging</name>
-                    <url>https://oss.sonatype.org/content/repositories/staging</url>
+                    <url>https://jakarta.oss.sonatype.org/content/repositories/staging</url>
```

Comment:
Same comment as above... Any reason why we shouldn't use the existing sonatypeOssDistMgmtSnapshotsUrl property?